### DISCUSSION TITLE
[質問]: スワイプで削除機能をつけるためのコードを書いたのですが、スワイプができないです。

### DISCUSSION THREAD

**Discussion by sheep1857** [lit-iphone-course/lit-iphone-course#146] [Question-original] [answered]
### エラーの概要

サイトに沿ってスワイプをしたらDeliteボタンが出てくるようにコードを記入したのですが、スワイプができません。

### エラーが起こるタイミング

Table Viewのセルをスワイプしようとしたとき

### エラーが起こる場所

ViewController

### 想定している挙動

左にスワイプするとDeliteボタンが出てくる

### 原因だと思われる箇所(render: swift)

<img width="765" alt="スクリーンショット 2023-02-26 13 24 06" src="https://user-images.githubusercontent.com/105029351/221392342-5c4a8e4d-f1ee-4494-9252-f6d2df3bb4d5.png">

### 自分で調べた資料１

https://qiita.com/tanakadaichi_1989/items/dacd090f0e4d3ab145e8

### GitHubのURL

https://github.com/sheep1857/Todo.git

### TODOリスト

- [X] GitHubに最新のコードをアップロードしました。
- [X] できるだけ具体的なタイトルを設定しました。

**Comment by txxxxc** [comment, marked as answer]
丁寧に質問ありがとうー！
GitHubにあげてもらっているzipに、`xcodeproj`のファイルしか含まれていないみたいです！
こちら側でデバッグするために、ソースコード全体を送ってほしいです！(Slackで送ってもらっても構いません！)

<img width="780" alt="image" src="https://user-images.githubusercontent.com/22112440/221398313-bd631057-220a-4ec2-b9c4-5cbd1dbe7c03.png">
----

ソースコードを確認していないので、予想になっちゃうんだけどviewDidLoadの中にこれって書いてる？↓

```swift
tableView.delegate = self
```

確認お願いしますー！

**Reply by sheep1857** [reply]
すみません、ソースコード入れ直しました！
viewDidLoadにviewDidLoadにその文書いてなかった為書いて実行してみたところ、無事スワイプできました。ありがとうございます！

**Reply by txxxxc** [reply]
ありがとう！！
良かったですー🙌🙌🙌
